Sources/Framework/Models/Answer.swift

```diff
@@ -16,6 +8,8 @@ public enum AnswerType: String {
 
 }
 
+// swiftlint:disable no_extension_access_modifier
```

Comment:
Please delete disabling this rule. Keyword public should be used here:

```swift
extension AnswerType {

      public static var allTypes: [AnswerType] {
...
```

instead of here:

```swift
```swift
public extension AnswerType {

      static var allTypes: [AnswerType] {
...
```
```